ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -169,34 +196,101 @@ tests:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
 - as: aws-ipi-private-f28
-  cron: 36 12 9 * *
+  cron: 40 12 6 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-private
-- as: aws-ipi-proxy-sts-f14
-  cron: 4 18 10,25 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f28
+  cron: 24 4 8 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: aws-ipi-proxy-sts-f28
+  cron: 27 11 15 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-registry-s3-cloudfront-f28
+  cron: 5 14 9 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-registry-s3-cloudfront
+- as: aws-ipi-rm-root-secret-f28
+  cron: 10 8 21 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-rm-root-secret
+- as: aws-ipi-shared-vpc-phz-f28
+  cron: 36 4 12 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-shared-vpc-phz-sts-fips-f28
+  cron: 51 7 19 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts
+- as: aws-sc2s-ipi-disc-priv-fips-f28
```

Comment:
This install profile was introduced since 4.12, right?

Reply:
it was introduced in 4.11 https://docs.openshift.com/container-platform/4.11/installing/installing_aws/installing-aws-secret-region.html

@JianLi-RH A tip, if you don't know when the feature was introduced, you can refer to the official doc.

Reply:
I searched `aws-sc2s-ipi-disc-priv-fips` and it is new in 4.12, but yes, as you said `aws-sc2s` was introduced in 4.11:
![image](https://github.com/openshift/release/assets/83053288/ff3299bc-6b74-4d07-b036-9ed1db45e643)

Reply:
Let us follow the existing install workflows in source version, that can help team member easily follow the same rules.